active/gridsyntax/GridSyntaxSpec2.md

```diff
@@ -0,0 +1,368 @@
+{
+    ColumnDefinition();
+    Windows.UI.Xaml.GridLength Width;
+    Double MaxWidth;
+    Double MinWidth;
+    Double ActualWidth{ get; };
+    static Windows.UI.Xaml.DependencyProperty WidthProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxWidthProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MinWidthProperty{ get; };
+};
+```
+
+## RowDefinition API
+
+```csharp
+[webhosthidden]
+
+// vvv
+[contentproperty("Height")] // Sets the RowDefinition's content property to the Height property.
+// ^^^
+
+runtimeclass RowDefinition : Windows.UI.Xaml.DependencyObject
+{
+    RowDefinition();
+    Windows.UI.Xaml.GridLength Height;
+    Double MaxHeight;
+    Double MinHeight;
+    Double ActualHeight{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeightProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxHeightProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MinHeightProperty{ get; };
+};
+```
+
+# Appendix
+
+## Alternatives
+
+## Syntax Considerations and Rationale
+In developing the syntax, the team considered developer familiarty and patterns as much as possible to ensure that the Grid learning curve could be decreased. One of the considerations was how similar the syntax should be to JSON, as that's a popular notation style that often addresses the same problems that XAML does. 
+
+JSON defines their arrays as: ```"words" : ["apple", "banana", "cherry"]```
+
+XAML defines a collection-typed property as: ```<Words="'Apple', 'Banana', 'Cherry'"/>```
+
+The syntax that we chose encloses strings individually in quotes, as JSON does, but it does so in a way that is more consistent with existing XAML style and XAML functions. An example of this syntax used elsewhere in XAML is for the x:Bind syntax:
+
+```xml <TextBlock Text="{x:Bind VM.MyText, FallbackValue='Hello, world', Mode=TwoWay}" />```
+
+This uses a comma as a delimiter, but also allows strings to have commas within them. For these reasons, we chose to use the syntax in which a collection can be delimited by commas but have more complex values be encased within single quotes.
+
+
+## Release Details
+The xmlns will not be updated for this XAML language feature addition, i.e. there will be no formal update to the XAML language. Visual Studio, however, will have to be updated to accept the new syntax structure. This feature will be released with WinUI and not with the XAML SDK.
+
+## Tooling/Visual Studio Considerations
+The Designer is able to tell when an element is createable from string, and this language feature will make it so all collection-type properties are createable from string (if that string is formatted as a comma-separated list). So, the Designer will recognize that a collection-type property is assigned to a string, check if it is assignable from string, and then attempt to treat the string as a comma-separated list, similar to the way the XAML Parser works. 
```

Comment:
I believe this isn't completely true anymore. If a collection-typed property defaults to null, it isn't able to be creatable from string via this new syntax.